adafruit_mcp3xxx/mcp3004.py

```diff
@@ -20,33 +20,42 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 """
-`adafruit_MCP3xxx`
-====================================================
+`mcp3004.py`
+================================================
+MCP3004 4-channel, 10-bit, analog-to-digital
+converter instance.
 
-.. todo:: Describe what the module does
-
-* Author(s): ladyada
-
-Implementation Notes
---------------------
-
-**Hardware:**
-
-.. todo:: Add links to any specific hardware product page(s), or category page(s). Use unordered list & hyperlink rST
-   inline format: "* `Link Text <url>`_"
-
-**Software and Dependencies:**
-
-* Adafruit CircuitPython firmware for the supported boards:
-  https://github.com/adafruit/circuitpython/releases
-  
-.. todo:: Uncomment or remove the Bus Device and/or the Register library dependencies based on the library's use of either.
-
-# * Adafruit's Bus Device library: https://github.com/adafruit/Adafruit_CircuitPython_BusDevice
-# * Adafruit's Register library: https://github.com/adafruit/Adafruit_CircuitPython_Register
+* Author(s): Brent Rubell
 """
 
-# imports
-
-__version__ = "0.0.0-auto.0"
-__repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_MCP3xxx.git"
+from .mcp3xxx import MCP3xxx
+
+class MCP3004(MCP3xxx):
+    """
+    MCP3004 Pin Mapping.
+    """
+    pin_0 = 0
+    pin_1 = 1
+    pin_2 = 2
+    pin_3 = 3
```

Comment:
I think these should be constants outside the class and be `P0`-`P3` to be similar to `D3`. I'd like us to move away from A and D prefixes since pins can do many things more often than not.

Reply:
I had them inside the class for tab completion -`MCP3008.P0`. Would they move outside the class but still within mcp3008/04?

Reply:
Why would these be moving them outside the class? Would Diff_Pins remain inside the mcp3008/04 class?

Reply:
I moved them outside in #f51f443, was this what you wanted?

Reply:
Yup! That looks right. I like it outside because we expect it to be used by user code. DIFF_PINS can stay inside since its only used internally by the library.